src/AddressCard.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import { bonds, InlineAccount, InlineBalance, AccountIcon, AddressLabel, AccountLabel, EtherBalance, CoinList } from 'parity-reactive-ui';
+import { Header, Card, Image, Table, Button, Popup, Input } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { DappLink } from '@parity/ui';
+import { BigNumber } from 'bignumber.js';
+import { Bond } from 'oo7';
+import { formatBalance } from 'oo7-parity';
+import { ReactiveComponent, Rspan, Rimg } from 'oo7-react';
+import _ from 'lodash';
```

Comment:
Considder importing the specific parts from lodash. Even better, use the single lodash packages, it does make a difference in the final compiled output.